source/_integrations/nest.markdown

```diff
@@ -299,6 +299,31 @@ All Google Nest Cam models and the Google Nest Hello Video Doorbell support [Dev
 
 See [Automating Home Assistant](/getting-started/automation/) for the getting started guide on automations or the [Automation](/docs/automation/) documentation for full details.
 
+### Example
+
+This automation will trigger when a `nest_event` event type with a type of `camera_motion` is received from the specified `device_id`.
+
+```yaml
+alias: motion alert
+description: ''
+trigger:
+  - platform: event
+    event_type: nest_event
+    event_data:
+      device_id: 37d6639fa63c348310d930825bb9e430
```

Comment:
Please use a placeholder; alternatively; add a YAML comment on that it needs to be replaced?

```suggestion
      device_id: YOUR_DEVICE_ID
```

Reply:
:+1: works for me, I've applied this change in a new commit. Thanks for your feedback!